From 28fc1d56333f9865535ed4fde6b779ee9cf44682 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:05:25 -0400
Subject: [powerpc] scom: CONFIG_SCOM_DEBUGFS should depend on CONFIG_DEBUG_FS

Message-id: <1410545655-205645-97-git-send-email-dzickus@redhat.com>
Patchwork-id: 94142
O-Subject: [RHEL7 PATCH 096/626] powerpc/scom: CONFIG_SCOM_DEBUGFS should depend on CONFIG_DEBUG_FS
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit 4777f79689bc722d12b3cfa1e2e92809cbe7bb38
Author: Benjamin Herrenschmidt <benh@kernel.crashing.org>
Date:   Thu Aug 29 16:58:12 2013 +1000

    powerpc/scom: CONFIG_SCOM_DEBUGFS should depend on CONFIG_DEBUG_FS

    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/sysdev/Kconfig b/arch/powerpc/sysdev/Kconfig
index ab4cb54..13ec968 100644
--- a/arch/powerpc/sysdev/Kconfig
+++ b/arch/powerpc/sysdev/Kconfig
@@ -28,7 +28,7 @@ config PPC_SCOM
 
 config SCOM_DEBUGFS
  bool "Expose SCOM controllers via debugfs"
- depends on PPC_SCOM
+ depends on PPC_SCOM && DEBUG_FS
  default n
 
 config GE_FPGA
-- 
1.7.1